Makefile

```diff
@@ -24,9 +24,12 @@ clean:
 	rm -f *deb *rpm
 	rm -f sha256sum md5sum
 
-$(NAME): $(wildcard **/*.go)
```

Comment:
May I ask, why?

Reply:
Rebuild not work when go files in root dir was changed. But it not solved a problem. Better way change to `$(NAME): $(wildcard **/*.go *.go)`

Reply:
You are right, but it won't work too, unfortunately. It should be replaced by `$(shell find . -name '*.go')`. It still won't find files, if they contain spaces, but it's already good enough